elements/lisk-cryptography/src/ed.ts

```diff
@@ -26,29 +25,15 @@ import {
 	verifyDetached,
 } from './nacl';
 
+const MESSAGE_TAG_NON_PROTOCOL_MESSAGE = 'LSK_NPM_';
```

Comment:
This would be better to be exposed from constants if someone wants to verify the message manually. (also if we expose the generic verify, this can be used by other libraries)